node/cli/src/command.rs

```diff
@@ -69,8 +68,11 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 		"frequency-bench" => return Ok(Box::new(chain_spec::frequency::benchmark_mainnet_config())),
 		#[cfg(feature = "frequency")]
 		"frequency" => return Ok(Box::new(chain_spec::frequency::load_frequency_spec())),
+		#[cfg(feature = "frequency-no-relay")]
+		"dev" | "frequency-no-relay" =>
+			return Ok(Box::new(chain_spec::frequency_rococo::development_config())),
 		#[cfg(feature = "frequency-rococo-local")]
-		"frequency-local" | "dev" =>
+		"frequency-local" | "frequency-rococo-local" =>
```

Comment:
It's a name that predates this PR and I have been hesitant to remove it because it's used in CI, api-augment and other places.  Perhaps, it's better done in a new issue/pr?  What do you think?

Reply:
Let's refactor this in a separate issue/pr.